mmv1/api/type.rb

```diff
@@ -755,6 +765,52 @@ def exclude_if_not_in_version!(version)
     # simpler property to generate and means we can avoid conditional logic
     # in Map.
     class KeyValuePairs < Composite
+      # Ignore writing the "effectcive_labels" and "effective_annotations" fields to API.
+      attr_reader :ignore_write
```

Comment:
stray thought: Longer-term, we could evaluate merging the implementation of this field and virtual fields as they're similar-ish in concept

Reply:
I will wait until `terraform_labels` is added later for the merging.